### ISSUE TITLE
2025-02-13, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#82]
# Today's Arxiv Papers

(1) [Utility Engineering: Analyzing and Controlling Emergent Value Systems in AIs](http://arxiv.org/abs/2502.08640v1), 2025-02-12

(2) [Examining Multilingual Embedding Models Cross-Lingually Through LLM-Generated Adversarial Examples](http://arxiv.org/abs/2502.08638v1), 2025-02-12

(3) [Ensemble based approach to quantifying uncertainty of LLM based classifications](http://arxiv.org/abs/2502.08631v1), 2025-02-12

(4) [SPeCtrum: A Grounded Framework for Multidimensional Identity Representation in LLM-Based Agent](http://arxiv.org/abs/2502.08599v1), 2025-02-12

(5) [Commercial LLM Agents Are Already Vulnerable to Simple Yet Dangerous Attacks](http://arxiv.org/abs/2502.08586v1), 2025-02-12

(6) [Mapping the Landscape of Generative AI in Network Monitoring and Management](http://arxiv.org/abs/2502.08576v1), 2025-02-12

(7) [QA-Expand: Multi-Question Answer Generation for Enhanced Query Expansion in Information Retrieval](http://arxiv.org/abs/2502.08557v1), 2025-02-12

(8) [Fostering Appropriate Reliance on Large Language Models: The Role of Explanations, Sources, and Inconsistencies](http://dx.doi.org/10.1145/3706598.3714020), 2025-02-12

(9) [LLMs can implicitly learn from mistakes in-context](http://arxiv.org/abs/2502.08550v1), 2025-02-12

(10) [LLM Pretraining with Continuous Concepts](http://arxiv.org/abs/2502.08524v1), 2025-02-12

(11) [The Paradox of Stochasticity: Limited Creativity and Computational Decoupling in Temperature-Varied LLM Outputs of Structured Fictional Data](http://arxiv.org/abs/2502.08515v1), 2025-02-12

(12) [Faithful, Unfaithful or Ambiguous? Multi-Agent Debate with Initial Stance for Summary Evaluation](http://arxiv.org/abs/2502.08514v1), 2025-02-12

(13) [Measuring Diversity in Synthetic Datasets](http://arxiv.org/abs/2502.08512v1), 2025-02-12

(14) [Explanation based In-Context Demonstrations Retrieval for Multilingual Grammatical Error Correction](http://arxiv.org/abs/2502.08507v1), 2025-02-12

(15) [Revisiting 3D LLM Benchmarks: Are We Really Testing 3D Capabilities?](http://arxiv.org/abs/2502.08503v1), 2025-02-12

(16) [Salamandra Technical Report](http://arxiv.org/abs/2502.08489v1), 2025-02-12

(17) [mmE5: Improving Multimodal Multilingual Embeddings via High-quality Synthetic Data](http://arxiv.org/abs/2502.08468v1), 2025-02-12

(18) [Better Embeddings with Coupled Adam](http://arxiv.org/abs/2502.08441v1), 2025-02-12

(19) [From Haystack to Needle: Label Space Reduction for Zero-shot Classification](http://arxiv.org/abs/2502.08436v1), 2025-02-12

(20) [A Semantic Parsing Algorithm to Solve Linear Ordering Problems](http://arxiv.org/abs/2502.08415v1), 2025-02-12

(21) [IssueBench: Millions of Realistic Prompts for Measuring Issue Bias in LLM Writing Assistance](http://arxiv.org/abs/2502.08395v1), 2025-02-12

(22) [ViLa-MIL: Dual-scale Vision-Language Multiple Instance Learning for Whole Slide Image Classification](http://arxiv.org/abs/2502.08391v1), 2025-02-12

(23) [The MoE-Empowered Edge LLMs Deployment: Architecture, Challenges, and Opportunities](http://arxiv.org/abs/2502.08381v1), 2025-02-12

(24) [Top-Theta Attention: Sparsifying Transformers by Compensated Thresholding](http://arxiv.org/abs/2502.08363v1), 2025-02-12

(25) [Systematic Knowledge Injection into Large Language Models via Diverse Augmentation for Domain-Specific RAG](http://arxiv.org/abs/2502.08356v1), 2025-02-12

(26) [Trustworthy GNNs with LLMs: A Systematic Review and Taxonomy](http://arxiv.org/abs/2502.08353v1), 2025-02-12

(27) [Graph Foundation Models for Recommendation: A Comprehensive Survey](http://arxiv.org/abs/2502.08346v1), 2025-02-12

(28) [Modification and Generated-Text Detection: Achieving Dual Detection Capabilities for the Outputs of LLM by Watermark](http://arxiv.org/abs/2502.08332v1), 2025-02-12

(29) [Contextual Compression Encoding for Large Language Models: A Novel Framework for Multi-Layered Parameter Space Pruning](http://arxiv.org/abs/2502.08323v1), 2025-02-12

(30) [MultiProSE: A Multi-label Arabic Dataset for Propaganda, Sentiment, and Emotion Detection](http://arxiv.org/abs/2502.08319v1), 2025-02-12

(31) [Word Synchronization Challenge: A Benchmark for Word Association Responses for LLMs](http://arxiv.org/abs/2502.08312v1), 2025-02-12

(32) [Unlocking Scaling Law in Industrial Recommendation Systems with a Three-step Paradigm based Large User Model](http://arxiv.org/abs/2502.08309v1), 2025-02-12

(33) [Compromising Honesty and Harmlessness in Language Models via Deception Attacks](http://arxiv.org/abs/2502.08301v1), 2025-02-12

(34) [Improving Existing Optimization Algorithms with LLMs](http://arxiv.org/abs/2502.08298v1), 2025-02-12

(35) [Redefining Simplicity: Benchmarking Large Language Models from Lexical to Document Simplification](http://arxiv.org/abs/2502.08281v1), 2025-02-12

(36) [MoLoRec: A Generalizable and Efficient Framework for LLM-Based Recommendation](http://arxiv.org/abs/2502.08271v1), 2025-02-12

(37) [Exploring the Potential of Large Language Models to Simulate Personality](http://arxiv.org/abs/2502.08265v1), 2025-02-12

(38) [FixDrive: Automatically Repairing Autonomous Vehicle Driving Behaviour for $0.08 per Violation](http://arxiv.org/abs/2502.08260v1), 2025-02-12

(39) [Learning Human Skill Generators at Key-Step Levels](http://arxiv.org/abs/2502.08234v1), 2025-02-12

(40) [Flow-of-Action: SOP Enhanced LLM-Based Multi-Agent System for Root Cause Analysis](http://arxiv.org/abs/2502.08224v1), 2025-02-12

(41) [LLM Modules: Knowledge Transfer from a Large to a Small Model using Enhanced Cross-Attention](http://arxiv.org/abs/2502.08213v1), 2025-02-12

(42) [Memory Offloading for Large Language Model Inference with Latency SLO Guarantees](http://arxiv.org/abs/2502.08182v1), 2025-02-12

(43) [Enhancing LLM Character-Level Manipulation via Divide and Conquer](http://arxiv.org/abs/2502.08180v1), 2025-02-12

(44) [ParetoRAG: Leveraging Sentence-Context Attention for Robust and Efficient Retrieval-Augmented Generation](http://arxiv.org/abs/2502.08178v1), 2025-02-12

(45) [SycEval: Evaluating LLM Sycophancy](http://arxiv.org/abs/2502.08177v1), 2025-02-12

(46) [Intention is All You Need: Refining Your Code from Your Intention](http://arxiv.org/abs/2502.08172v1), 2025-02-12

(47) [ACCESS : A Benchmark for Abstract Causal Event Discovery and Reasoning](http://arxiv.org/abs/2502.08148v1), 2025-02-12

(48) [Democratizing AI: Open-source Scalable LLM Training on GPU-based Supercomputers](http://arxiv.org/abs/2502.08145v1), 2025-02-12

(49) [Bridging the Safety Gap: A Guardrail Pipeline for Trustworthy LLM Inferences](http://arxiv.org/abs/2502.08142v1), 2025-02-12

(50) [LowRA: Accurate and Efficient LoRA Fine-Tuning of LLMs under 2 Bits](http://arxiv.org/abs/2502.08141v1), 2025-02-12

(51) [Selective Self-to-Supervised Fine-Tuning for Generalization in Large Language Models](http://arxiv.org/abs/2502.08130v1), 2025-02-12

(52) [Fino1: On the Transferability of Reasoning Enhanced LLMs to Finance](http://arxiv.org/abs/2502.08127v1), 2025-02-12

(53) [HuDEx: Integrating Hallucination Detection and Explainability for Enhancing the Reliability of LLM responses](http://arxiv.org/abs/2502.08109v1), 2025-02-12

(54) [NLI under the Microscope: What Atomic Hypothesis Decomposition Reveals](http://arxiv.org/abs/2502.08080v1), 2025-02-12

(55) [Large language models perpetuate bias in palliative care: development and analysis of the Palliative Care Adversarial Dataset (PCAD)](http://arxiv.org/abs/2502.08073v1), 2025-02-12

(56) [On Mechanistic Circuits for Extractive Question-Answering](http://arxiv.org/abs/2502.08059v1), 2025-02-12

(57) [Break the Checkbox: Challenging Closed-Style Evaluations of Cultural Alignment in LLMs](http://arxiv.org/abs/2502.08045v1), 2025-02-12

(58) [Franken-Adapter: Cross-Lingual Adaptation of LLMs by Embedding Surgery](http://arxiv.org/abs/2502.08037v1), 2025-02-12

(59) [Contextual Subspace Manifold Projection for Structural Refinement of Large Language Model Representations](http://arxiv.org/abs/2502.08026v1), 2025-02-12

(60) [Speculate, then Collaborate: Fusing Knowledge of Language Models during Decoding](http://arxiv.org/abs/2502.08020v1), 2025-02-11

(61) [The Geometry of Prompting: Unveiling Distinct Mechanisms of Task Adaptation in Language Models](http://arxiv.org/abs/2502.08009v1), 2025-02-11

(62) [An Interactive Framework for Implementing Privacy-Preserving Federated Learning: Experiments on Large Language Models](http://arxiv.org/abs/2502.08008v1), 2025-02-11

(63) [Universal Adversarial Attack on Aligned Multimodal LLMs](http://arxiv.org/abs/2502.07987v1), 2025-02-11

(64) [Deep Semantic Graph Learning via LLM based Node Enhancement](http://arxiv.org/abs/2502.07982v1), 2025-02-11

(65) [AI Humor Generation: Cognitive, Social and Creative Skills for Effective Humor](http://arxiv.org/abs/2502.07981v1), 2025-02-11

(66) [CIRCUIT: A Benchmark for Circuit Interpretation and Reasoning Capabilities of LLMs](http://arxiv.org/abs/2502.07980v1), 2025-02-11

(67) [From Hazard Identification to Controller Design: Proactive and LLM-Supported Safety Engineering for ML-Powered Systems](http://arxiv.org/abs/2502.07974v1), 2025-02-11

(68) [Caught in the Web of Words: Do LLMs Fall for Spin in Medical Literature?](http://arxiv.org/abs/2502.07963v1), 2025-02-11

(69) [Accelerating Scientific Research Through a Multi-LLM Framework](http://arxiv.org/abs/2502.07960v1), 2025-02-11

(70) [Bridging HCI and AI Research for the Evaluation of Conversational SE Assistants](http://arxiv.org/abs/2502.07956v1), 2025-02-11

(71) [Symbiotic Cooperation for Web Agents: Harnessing Complementary Strengths of Large and Small LLMs](http://arxiv.org/abs/2502.07942v1), 2025-02-11

(72) [Distributed Approach to Haskell Based Applications Refactoring with LLMs Based Multi-Agent Systems](http://arxiv.org/abs/2502.07928v1), 2025-02-11

(73) [Sign Operator for Coping with Heavy-Tailed Noise: High Probability Convergence Bounds with Extensions to Distributed Optimization and Comparison Oracle](http://arxiv.org/abs/2502.07923v1), 2025-02-11

(74) [Elevating Legal LLM Responses: Harnessing Trainable Logical Structures and Semantic Knowledge with Legal Reasoning](http://arxiv.org/abs/2502.07912v1), 2025-02-11

(75) [DeepSeek on a Trip: Inducing Targeted Visual Hallucinations via Representation Vulnerabilities](http://arxiv.org/abs/2502.07905v1), 2025-02-11

(76) [Intelligent Legal Assistant: An Interactive Clarification System for Legal Question Answering](http://arxiv.org/abs/2502.07904v1), 2025-02-11

(77) [HexGen-2: Disaggregated Generative Inference of LLMs in Heterogeneous Environment](http://arxiv.org/abs/2502.07903v1), 2025-02-11

(78) [DarwinLM: Evolutionary Structured Pruning of Large Language Models](http://arxiv.org/abs/2502.07780v1), 2025-02-11

(79) [Auditing Prompt Caching in Language Model APIs](http://arxiv.org/abs/2502.07776v1), 2025-02-11

(80) [Automatic Robot Task Planning by Integrating Large Language Model with Genetic Programming](http://arxiv.org/abs/2502.07772v1), 2025-02-11

(81) [Breaking Down Bias: On The Limits of Generalizable Pruning Strategies](http://arxiv.org/abs/2502.07771v1), 2025-02-11

(82) [Great Power Brings Great Responsibility: Personalizing Conversational AI for Diverse Problem-Solvers](http://arxiv.org/abs/2502.07763v1), 2025-02-11

(83) [Scalable Fingerprinting of Large Language Models](http://arxiv.org/abs/2502.07760v1), 2025-02-11

(84) [Towards Efficient Optimizer Design for LLM via Structured Fisher Approximation with a Low-Rank Extension](http://arxiv.org/abs/2502.07752v1), 2025-02-11

(85) [TransMLA: Multi-head Latent Attention Is All You Need](http://arxiv.org/abs/2502.07864v1), 2025-02-11

(86) [WHODUNIT: Evaluation benchmark for culprit detection in mystery stories](http://arxiv.org/abs/2502.07747v1), 2025-02-11

(87) [The Economics of Large Language Models: Token Allocation, Fine-Tuning, and Optimal Pricing](http://arxiv.org/abs/2502.07736v1), 2025-02-11

(88) [Economics of Sourcing Human Data](http://arxiv.org/abs/2502.07732v1), 2025-02-11

(89) [Verifying LLM-Generated Code in the Context of Software Verification with Ada/SPARK](http://arxiv.org/abs/2502.07728v1), 2025-02-11

(90) [BalanceKV: KV Cache Compression through Discrepancy Theory](http://arxiv.org/abs/2502.07861v1), 2025-02-11

(91) [MAGELLAN: Metacognitive predictions of learning progress guide autotelic LLM agents in large goal spaces](http://arxiv.org/abs/2502.07709v2), 2025-02-11

(92) [A Framework for LLM-powered Design Assistants](http://arxiv.org/abs/2502.07698v1), 2025-02-11

(93) [Large Language Models as Proxies for Theories of Human Linguistic Cognition](http://arxiv.org/abs/2502.07687v1), 2025-02-11

(94) [Auto-Drafting Police Reports from Noisy ASR Outputs: A Trust-Centered LLM Approach](http://arxiv.org/abs/2502.07677v2), 2025-02-11

(95) [SymGPT: Auditing Smart Contracts via Combining Symbolic Execution with Large Language Models](http://arxiv.org/abs/2502.07644v2), 2025-02-11

(96) [FoQA: A Faroese Question-Answering Dataset](http://arxiv.org/abs/2502.07642v1), 2025-02-11

(97) [Goedel-Prover: A Frontier Model for Open-Source Automated Theorem Proving](http://arxiv.org/abs/2502.07640v1), 2025-02-11

(98) [Exploring Mobile Touch Interaction with Large Language Models](http://dx.doi.org/10.1145/3706598.3713554), 2025-02-11

(99) [Beyond Prompting: Time2Lang -- Bridging Time-Series Foundation Models and Large Language Models for Health Sensing](http://arxiv.org/abs/2502.07608v2), 2025-02-11

(100) [Towards Zero-Shot Anomaly Detection and Reasoning with Multimodal Large Language Models](http://arxiv.org/abs/2502.07601v1), 2025-02-11
